lib/msal-node/package.json

```diff
@@ -28,9 +30,10 @@
     "start": "tsdx watch",
     "build": "tsdx build",
     "test": "tsdx test .*.spec.*",
-    "test:watch": "tsdx test .*.spec.* --watch",
-    "test:coverage": "tsdx test .*.spec.* --coverage",
-    "lint": "tsdx lint",
+    "test:n": "tsdx test",
+    "test:w": "tsdx test .*.spec.* --watch",
+    "test:c": "tsdx test .*.spec.* --coverage",
+    "lint": "tsdx lint --fix",
```

Comment:
I would argue to lint be its own script: `"lint:fix": "npm run lint -- --fix"`

Reply:
yes, did this as it was annoying me, will do.

Reply:
Done